odoo/service/db.py

```diff
@@ -114,11 +114,17 @@ def _create_empty_database(name):
                 sql.Identifier(name), collate, sql.Identifier(chosen_template)
             ))
 
-    if odoo.tools.config['unaccent']:
+    if odoo.tools.config['unaccent']:  # TODO: Should removed this config parameters, trigram extention is done differently ?
```

Comment:
This was made to automatically install the extension on all databases installed if `--unaccent` is enabled. That seems like a useful feature.

Trigram requires enabling the extension manually. If there's something to change, it certainly isn't the ability to enable unaccent by default is it?

Reply:
Yes, you are right, I put this TODO because things are done differently for trigram extension (now install on fly if you have access) vs unaccent extension (install when you create a new DB). Maybe we should add a `--trigram` to have consistency between both extension management ?

Reply:
Possibly. Or keep the legacy `unaccent` but add a `--extensions` option which can take multiple extensions which it tries to install? That way it works if / when we decide to enable, say, `citext`, or `fuzzystrmatch`, or `isn`.